### DISCUSSION TITLE
Any plan for bar chart or line chart

### DISCUSSION THREAD

**Discussion by Lukexcc** [mermaid-js/mermaid#4270] [Ideas]
Hi folks, is there any plan for bar chart or line chart?
I think these two kind charts are very common and it's not supported. Correct me if Im wrong.

**Comment by ghost** [comment]
Duplicate of: [line-graph in mermaid-js?](https://github.com/orgs/mermaid-js/discussions/3522)

**Reply by sidharthv96** [reply]
Being worked on at https://github.com/mermaid-js/mermaid/issues/4269

**Comment by sidharthv96** [comment]
https://github.com/mermaid-js/mermaid/issues/4269
